hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledExecutorDataSerializerHook.java

```diff
@@ -150,6 +156,10 @@ public IdentifiedDataSerializable create(int typeId) {
                         return new GetAllScheduledOnPartitionOperation();
                     case GET_ALL_SCHEDULED_ON_PARTITION_OPERATION_FACTORY:
                         return new GetAllScheduledOnPartitionOperationFactory();
+                    case SPLIT_BRAIN_MERGE_OP:
+                        return new SplitBrainMergeOperation();
+                    case SPLIT_BRAIN_SYNC_MERGED_OP:
+                        return new SplitBrainSyncMergedBackupOperation();
```

Comment:
can we use `SPLIT_BRAIN_MERGE_BACKUP_OP ` and `SplitBrainMergeBackupOperation` instead to inline naming with the other services in the code base?